.github/workflows/testing.yml

```diff
@@ -32,6 +32,10 @@ jobs:
           key: vscode-ripgrep-bin-${{ runner.os }}-${{ runner.arch }}-${{ hashFiles('yarn.lock') }}
           path: node_modules/@vscode/ripgrep/bin/
 
+      - name: Install all npm packages (libs/play)
+        working-directory: libs/play
+        run: npm ci
```

Comment:
Why is this necessary, given that `yarn install` should also run `npm install` there via the `prepare` script?

https://github.com/mdn/yari/blob/f19459869e3c1d0e1f9210f0bac57da3d1a5d398/package.json#L35-L40

Reply:
Ah right, it was an ordering issue: the install happened in `cloud-function` first, before `play`, and so `play` was copied sans `node_modules`: adding `@yari-internal/play` as a dep in `cloud-function` seems to hoist its dependencies into `cloud-functions`' lockfile, avoiding that problem